src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -157,11 +157,7 @@ protected FluidAttributes(Builder builder)
         this.density = builder.density;
         this.isGaseous = builder.isGaseous;
         this.rarity = builder.rarity;
-    }
-
-    public final String getName()
-    {
-        return this.fluidName;
```

Comment:
to avoid a breaking change here, can this become
```java
return owner.getRegistryName().toString();
```
and deprecate it?

Reply:
Do we actually have an RB at this point? I think we're early enough in dev that we shouldn't have to overly worry about deprecation. I'd rather it break and people use it correctly.

Reply:
That is my thinking. On top of that, this PR would have breaking changes regardless, so the value of this deprecated method that I just added last week seems too low to be worth keeping it.

I was hoping to get this merged quickly to avoid a lot of people having to deal with re-releasing updates, but I really want Lex's input on this.

Reply:
Yeah, if changes are *already* broken then I say maximize all possible breakage now. Let's make 1.14 as clean as possible. No deprecations, etc. Damn the torpedoes.

Reply:
Makes sense to me

Reply:
We are holding off a RB explicitly for this fluid system, so that it can do everything correctly and THEN we can stabilize the binary version. So once this is done, we'll worry about compat.